Makefile

```diff
@@ -383,3 +383,24 @@ gen_genesis_and_config:
 ## Clear the genesis and config files for LocalNet
 clear_genesis_and_config:
 	rm build/config/gen.*.json
+
+.PHONY: check_cross_module_imports
```

Comment:
No-op.

In @okdas's check on linters, we have the ability to create custom linters in go: https://github.com/pokt-network/pocket/pull/190

I think we might be able to port this there.